### PR TITLE
Fix image orientation issue

### PR DESCRIPTION
**Pull request by qianyanseu**
This pull request fixed issue #5 (image orientation).

### CODE REVIEW

File: JPSImagePickerController/JPSImagePickerController.m [hunk 1]
```diff
@@ -61,6 +61,53 @@ - (void)viewDidLoad {
     [self addCancelButton];
     [self addFlashButton];
     [self addCameraSwitchButton];
+
+
+
+    self.motionManager = [[CMMotionManager alloc] init];
```
Comment (jpsim) [inline JPSImagePickerController/JPSImagePickerController.m]: This should probably be in `viewDidAppear:` to match the `stopAccelerometerUpdates` call in `viewWillDisappear`.

File: JPSImagePickerController/JPSImagePickerController.m [hunk 2]
```diff
@@ -108,12 +158,12 @@ - (void)setFlashlightEnabled:(BOOL)flashlightEnabled {
 #pragma mark - Other
 
 - (void)updateFlashlightState {
-    
+
     if (![self currentDevice]) return;
-    
-    NSString *flashlightButtonTitle = self.isFlashlightEnabled ? @" On" : @" Off";
+
+    NSString *flashlightButtonTitle = self.isFlashlightEnabled ? @" ON" : @" OFF";
```
Comment (jpsim) [inline JPSImagePickerController/JPSImagePickerController.m]: These shouldn't be all-caps.

File: JPSImagePickerController/JPSImagePickerController.m [hunk 3]
```diff
@@ -506,10 +561,10 @@ - (void)addUseButton {
     self.useButton = [UIButton buttonWithType:UIButtonTypeSystem];
     self.useButton.titleLabel.font = [UIFont systemFontOfSize:18.0f];
     self.useButton.translatesAutoresizingMaskIntoConstraints = NO;
-    [self.useButton setTitle:@"Use" forState:UIControlStateNormal];
+    [self.useButton setTitle:@"Use Photo" forState:UIControlStateNormal];
```
Comment (jpsim) [inline JPSImagePickerController/JPSImagePickerController.m]: This used to be just "Use".

File: JPSImagePickerController/JPSImagePickerController.m [hunk 4]
```diff
@@ -665,29 +720,29 @@ - (NSUInteger)supportedInterfaceOrientations {
     return UIInterfaceOrientationMaskPortrait;
 }
 
-+ (UIImageOrientation)currentImageOrientation {
++ (UIImageOrientation)currentImageOrientation:(UIDeviceOrientation)deviceOrientation {
     // This is weird, but it works
     // By all means fix it, but make sure to test it afterwards
-    UIDeviceOrientation deviceOrientation = [[UIDevice currentDevice] orientation];
+    //UIDeviceOrientation deviceOrientation = deviceOrientation/*[[UIDevice currentDevice] orientation]*/;
```
Comment (jpsim) [inline JPSImagePickerController/JPSImagePickerController.m]: please just delete this line.

**Comment by jpsim** [comment]
Thanks for this, it bugged me when I wrote it, but I haven't used this code in a long time. Just ping me when you've addressed my feedback.

File: JPSImagePickerController/JPSImagePickerController.m (hunk 1 above)
Comment (qianyanseu) [inline JPSImagePickerController/JPSImagePickerController.m]: Good point!

**Comment by qianyanseu** [comment]
Cleaned up the code as you suggested. I'll close this one and open another pull request.

**Pull request closed without merging**